### PR TITLE
replace "analysis" with "asset" in asset-paths doc

### PR DESCRIPTION
**Pull request by kopackiw**
## Description & motivation

Found that documentation on `asset-paths` refers to `analysis-path`, but there is no relation between those two, so I assumed that this is a typo.

No open issue found.

### CODE REVIEW

File: website/docs/reference/project-configs/asset-paths.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Optionally specify a custom list of directories to copy to the `target` director
 </Changelog>
 
 ## Default
-By default, dbt will not compile any additional files as part of `docs generate`, i.e. `analysis-paths: []`
+By default, dbt will not compile any additional files as part of `docs generate`, i.e. `asset-paths: []`
```
Comment (mirnawong1) [inline website/docs/reference/project-configs/asset-paths.md]: thanks for opening this up @kopackiw! i believe what the docs is saying is that your analyses (saved in `analysis-path`) will not be generated by a `dbt docs generate`. I've suggested the below to clarify a bit further:

```suggestion
By default, dbt will not copy any additional files as part of docs generate (for example `asset-paths: []`). 
```

Comment (mirnawong1) [reply website/docs/reference/project-configs/asset-paths.md]: hey @b-per does this look right to you?

Comment (b-per) [reply website/docs/reference/project-configs/asset-paths.md]: I think that the issue that is being reported is that the `Default` section of the `asset-paths` page talks about `analysis-paths`.

I'd change it to: 
By default, dbt will not copy any additional files as part of `docs generate`, i.e. `asset-paths: []`

Replacing `copy` by `compile` and using the correct config item.

**Review by b-per — DISMISSED** [review]

**Review by mirnawong1 — DISMISSED** [review]
thanks for opening this up @kopackiw and @b-per for the feedback! made those changes for approval. does this look right now @b-per ?

**Comment by mirnawong1** [comment]
closes same pr contribution #2574 from @cedwards036

File: website/docs/reference/project-configs/asset-paths.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Optionally specify a custom list of directories to copy to the `target` director
 </Changelog>
 
 ## Default
-By default, dbt will not compile any additional files as part of `docs generate`, i.e. `analysis-paths: []`
+By default, dbt will not copy any additional files as part of docs generate (for example `asset-paths: []`).
```
Comment (mirnawong1) [inline website/docs/reference/project-configs/asset-paths.md]:
```suggestion
By default, dbt will not copy any additional files as part of docs generate, i.e. `asset-paths: []`
```

**Review by mirnawong1 — APPROVED** [review]

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
closes #2574

**Pull request merged**